compiler/noirc_evaluator/src/ssa/opt/constant_folding.rs

```diff
@@ -88,15 +80,8 @@ impl Ssa {
             };
         }
 
-        let brillig_info = Some(BrilligInfo { brillig, brillig_functions: &brillig_functions });
-
         for function in self.functions.values_mut() {
-            // We have already performed our final Brillig generation, so constant folding
-            // Brillig functions is unnecessary work.
-            if function.dfg.runtime().is_brillig() {
```

Comment:
I decided to run this over both runtimes as the compilation time benchmarks don't look to show many regressions. However, this may be a lot of unnecessary work as it diverges from the original purpose of `fold_constants_with_brillig` (folding Brillig entry points).